src/mon/Monitor.cc

```diff
@@ -1918,8 +1918,20 @@ void Monitor::win_election(epoch_t epoch, set<int>& active, uint64_t features,
   pending_metadata = metadata;
   outside_quorum.clear();
 
-  clog->info() << "mon." << name << "@" << rank
-		<< " won leader election with quorum " << quorum;
+  std::stringstream msg;
+  msg << "mon." << name << " is new leader, mons ";
+  for (auto i = quorum.begin(); i != quorum.end();) {
+    derr << "rank: " << *i << dendl;
```

Comment:
drop this

Reply:
also, see Monitor::get_quorum_names()?

Reply:
Thanks, I had a feeling I was forgetting about a helper